code/game/objects/items/storage/bags.dm

```diff
@@ -382,14 +382,14 @@
 	icon = 'icons/obj/chemical.dmi'
 	icon_state = "bag"
 	desc = "A bag for storing pills, patches, and bottles."
-	w_class = WEIGHT_CLASS_TINY
+	slot_flags = ITEM_SLOT_BELT|ITEM_SLOT_POCKET
 	resistance_flags = FLAMMABLE
 
 /obj/item/storage/bag/chemistry/ComponentInitialize()
 	. = ..()
 	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
-	STR.max_combined_w_class = 200
-	STR.max_items = 50
+	STR.storage_flags = STORAGE_FLAGS_VOLUME_DEFAULT
+	STR.max_volume = 100
```

Comment:
please put volumetric storage stuff in the defines file for sizes, with the old w_class system you're keeping track of 5 things
with volumetric storage you're keeping track of a big range of integers. i do not think there's anything to be lost by having it in one file for easy tweaks

Reply:
also how many units is this supposed to be?
i'd rather not have chemistry be too nerfed in terms of reagent capacity considering pill bottles are borderline meme right now with how weird it works due to the thresholding.